app/views/layouts/application.html.erb

```diff
@@ -12,7 +12,11 @@
     <nav class = "topnav">
       <%= link_to "Home", "/"%>
       <%= link_to "All Items", "/items"%>
-      <%= link_to "All Merchants", "/merchants"%>
+      <% if !current_admin? %>
+        <%= link_to "All Merchants", "/merchants"%>
+      <% else %>
+        <%= link_to "All Merchants", admin_merchants_path%>
+      <% end %>
```

Comment:
logged in as admin, this linked to /merchants and not /admin/merchants. the test was just navigating to /admin/merchants via `admin_merchants_path` but there was no link to it. so i added the logic

Reply:
Nice, I like that the same link shows up but it has a different path.

Reply:
yeah, it got me for a little bit. i was going to the normal /merchants and it's not a table, nothing to click on... but typing in /admin/merchants in the url and hitting enter worked. so it was a link/routes thing. i added both